docs/tutorials/tutorial_params_options_simulate.ipynb

```diff
@@ -0,0 +1,833 @@
+{
```

Comment:
<p>I think it would be nice here to discuss the simulated data a bit more. It might be helpful to discuss a bit what the different columns mean and maybe also discuss the indexing in a bit more detail (i.e. there is the <code>Identifier</code> level and <code>Period</code> level) since all of this carries over to more complicated models as well. E.g. here we are essentially simulating multiple Robinsons for a number of periods each. I remember somehow being a bit confused by this for the Robinson model specifically (but that might just be me).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
I can add that, it would be a good bridge to the how to simulate guide.